README.md

```diff
@@ -1,58 +1,173 @@
 # zola (né Gutenberg)
 
-[![Build Status](https://dev.azure.com/getzola/zola/_apis/build/status/getzola.zola?branchName=master)](https://dev.azure.com/getzola/zola/_build/latest?definitionId=1&branchName=master)
-
 A fast static site generator in a single binary with everything built-in.
 
-Documentation is available on [its site](https://www.getzola.org/documentation/getting-started/installation/) or
-in the `docs/content` folder of the repository and the community can use [its forum](https://zola.discourse.group).
+**zola version:** 0.13.0 ([CHANGELOG](CHANGELOG.md))
+
+**Test results:** [![Build Status](https://dev.azure.com/getzola/zola/_apis/build/status/getzola.zola?branchName=master)](https://dev.azure.com/getzola/zola/_build/latest?definitionId=1&branchName=master)
+
+**Minimum Supported Rust Version:** v1.44
```

Comment:
Can we remove that line or just point to the Azure CI file line? It's already in a few places and I always miss one when releasing a new version.